### DISCUSSION TITLE
Add ListBox direction

### DISCUSSION THREAD

**Discussion by jordisantamaria** [tailwindlabs/headlessui#3187] [Ideas]
Right now ListBox is showing content always on bottom, it should be able to show content on top when it doesn't fit on bottom.

**Comment by richardasymmetric** [comment]
100% agree. if the ListBox itself is rendered at the bottom of the page, the dropdown will appear off screen, and with the new anchor/modal thing adding `overflow: hidden` to the `html` element it means you can't access the dropdown properly
